JASP-Engine/JASP/R/reliabilityanalysis.R

```diff
@@ -194,20 +217,37 @@ ReliabilityAnalysis <- function(dataset = NULL, options, perform = "run",
 
 		footnotes <- .newFootnotes()
 
-		# is.null can be removed once options[["listwise"]] exists.
-		if (!is.null(options[["listwise"]])) {
-			exclwise = ifelse(options[["listwise"]], " listwise", " pairwise")
+		if (options[["missingValues"]] == "excludeCasesListwise") {
+			
+			exclwise = " listwise"
+			
 		} else {
-			exclwise = ""
+			
+			exclwise = " pairwise"
+			
 		}
 
-		nObs = nrow(dataset)
-		nExcluded = sum(!complete.cases(dataset))
-		nValid = nObs - nExcluded
+		nObs <- nrow(dataset)
+		nExcluded <- sum(!complete.cases(dataset))
+		nValid <- nObs - nExcluded
 
 		# message <- paste("Scale consists of items ", paste0(variables, collapse = ", "))
 		message <- sprintf("Of the observations, %d were used, %d were excluded%s, and %d were provided.",
 						   nValid, nExcluded, exclwise, nObs)
+		
+		if (options[["glbScale"]]) {
+			
+			if (length(variables) <= 2) {
+				
+				message <- paste(message, "Warning: Greatest lower bound can only be calculated for three or more variables.")
+				
+			} else if (isTryError(r[["glb"]])) {
+				
+				message <- paste(message, "Warning: Greatest lower bound could not be calculated.")
```

Comment:
Maybe it's also useful to include (a part of) the message the Psych package returns to the trycatch, to be as informative as possible.